gloo/docs/gloo_routing/virtual_services/security/ldap/_index.md

```diff
@@ -74,17 +72,12 @@ spec:
 EOF
 ```
 
-Now we can create a Virtual Service that routes any requests with the `/echo` prefix to the `echo` service.
+Now we can create a Virtual Service that routes any requests with the `/echo` prefix to the `http-echo` service.
 
-{{< tabs >}}
-{{< tab name="yaml" codelang="yaml">}}
+{{< highlight yaml "hl_lines=16-20" >}}
 {{< readfile file="gloo_routing/virtual_services/security/ldap/vs-echo-no-auth.yaml">}}
-{{< /tab >}}
-{{< tab name="glooctl" codelang="shell">}}
-glooctl create vs --name http-echo --namespace gloo-system
-glooctl add route --path-prefix /echo --dest-name default-http-echo-5678
-{{< /tab >}}
-{{< /tabs >}} 
```

Comment:
why drop the `glooctl` options?

Reply:
If we aren't going to keep copy-pasteable `glooctl` commands, then we should make the virtual service copy-pasteable (i.e., k apply -f - << EOF ...)

Reply:
We currently don't support the new auth config format in the CLI :/
I'll update to something copy-pasteable.